registry/storage-drivers/s3.md

```diff
@@ -31,7 +32,8 @@ Amazon S3 or S3 compatible services for object storage.
       <code>secretkey</code>
     </td>
     <td>
-      yes
+      yes, unless you use [IAM roles](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html).<br />
```

Comment:
It might just be better to say no, then have this IAM bit in the description field.